modules/oe_theme_content_organisation/oe_theme_content_organisation.module

```diff
@@ -56,10 +56,10 @@ function oe_theme_content_organisation_preprocess_oe_theme_helper_in_page_naviga
     return;
   }
 
-  $variables['image'] = ImageValueObject::fromStyledImageItem($thumbnail, 'oe_theme_ratio_3_2_medium');
+  $variables['image'] = ImageValueObject::fromStyledImageItem($thumbnail, 'oe_theme_medium_no_crop');
 
   $cacheability->addCacheableDependency($thumbnail->entity);
   $cacheability->addCacheableDependency(\Drupal::entityTypeManager()->getStorage('media_type')->load($media->bundle()));
-  $cacheability->addCacheableDependency(EntityViewDisplay::collectRenderDisplay($media, 'oe_theme_ratio_3_2_medium'));
+  $cacheability->addCacheableDependency(EntityViewDisplay::collectRenderDisplay($media, 'oe_theme_medium_no_crop'));
```

Comment:
This line doesn't make much sense. First of all, there is no media view mode called `oe_theme_medium_no_crop`. Nor was there one called `oe_theme_ratio_3_2_medium` so was wrong from before. This in fact will return the default entity view display of the media entity.

Second of all, why do we need to add this cache dependency? You are not using anything from the media entity display in building this image here.